src/slider.stories.ts

```diff
@@ -0,0 +1,283 @@
+import './icons/storybook.js';
+import { UPDATE_STORY_ARGS } from '@storybook/core-events';
+import { addons } from '@storybook/preview-api';
+import { html, nothing } from 'lit';
+import { unsafeHTML } from 'lit/directives/unsafe-html.js';
+import { withActions } from '@storybook/addon-actions/decorator';
+import type { Meta, StoryObj } from '@storybook/web-components';
+import SliderComponent from './slider.js';
+
+const meta: Meta = {
+  title: 'Slider',
+  decorators: [
+    withActions,
+    (story) =>
+      html`<form action="/">
+        <script type="ignore">
+          import '@crowdstrike/glide-core/slider.js';
+        </script>
+
+        ${story()}
+      </form>`,
+  ],
+  parameters: {
+    actions: {
+      handles: ['change', 'input', 'invalid'],
+    },
+    docs: {
+      story: {
+        autoplay: true,
+      },
+    },
+  },
+  args: {
+    label: 'Label',
+    'addEventListener(event, handler)': '',
+    'checkValidity()': '',
+    disabled: false,
+    'hide-label': false,
+    max: '',
+    min: '',
+    multiple: false,
+    name: '',
+    orientation: 'horizontal',
+    readonly: false,
+    'reportValidity()': '',
+    required: false,
+    'resetValidityFeedback()': '',
+    'setCustomValidity(message)': '',
+    'setValidity(flags, message)': '',
+    'slot="description"': '',
+    step: 1,
+    tooltip: '',
+    value: [],
+    version: '',
+  },
+  play(context) {
+    const slider = context.canvasElement.querySelector('glide-core-slider');
+
+    if (context.name.includes('Error') && slider instanceof SliderComponent) {
+      // Slider is a bit unique in that it always
```

Comment:
Nit. Mind making the width of the this comment similar to your other comments?

Reply:
Should be taken care of now. I added some rulers in my editor to help